nano/rpc_test/rpc.cpp

```diff
@@ -5592,40 +5592,6 @@ TEST (rpc, in_process)
 	ASSERT_EQ ("0", pending_text);
 }
 
-TEST (rpc_config, serialization)
```

Comment:
I think this test case should be converted to toml and kept.
I do not see an equivalent one for toml.

Reply:
There's never any serialization to TOML, deserialization is covered by `toml, rpc_config_deserialize_no_defaults`